java/opendcs/src/main/java/org/opendcs/database/impl/opendcs/OpenDcsPgProvider.java

```diff
@@ -32,7 +32,7 @@
  */
 public class OpenDcsPgProvider implements MigrationProvider
 {
-    private static final Logger log = LoggerFactory.getLogger(OpenDcsPgProvider.class);
+    private static final Logger log = LoggerFactory.getLogger(OpenDcsOracleProvider.class);
```

Comment:
```suggestion
    private static final Logger log = LoggerFactory.getLogger(OpenDcsPgProvider.class);
```
Trying to match the class name.